### PR TITLE
Update Node to 18.x

### PR DESCRIPTION
**Pull request by echaidemenos**
Updates node to v18.x and resolves any relevant incompatibilities

### CODE REVIEW

File: packages/website/src/common/StatusSnackbar/index.tsx [hunk 1]
```diff
@@ -54,11 +53,6 @@ const useStyles = makeStyles((theme: Theme) => ({
   alert: {
     alignItems: 'center',
   },
-  alertMessage: (hasMessage: boolean) => ({
```
Comment (ericboucher) [inline packages/website/src/common/StatusSnackbar/index.tsx:57]: why this change? was this not used anymore?

Comment (echaidemenos) [reply packages/website/src/common/StatusSnackbar/index.tsx:57]: This was wrongly typed, and i was having compilation errors. Re adding it :)

**Comment by ericboucher** [comment]
@echaidemenos any updates on the cloud functions?

**Review by ericboucher — APPROVED** [review]
